pkg/storage/migrator/migrator.go

```diff
@@ -189,7 +193,7 @@ func (m *Migrator) Migrate(num_blocks int) error {
 		m.wg.Add(1)
 
 		go func(block_no *storage.BlockInfo) {
-			err := m.migrateBlock(block_no.Block)
+			_, err := m.migrateBlock(block_no.Block)
```

Comment:
why are we returning the bytes here in this function if they're just ignored anyways?

Reply:
The return is used when called in the MigrateDirtyWithId() variant.
I've modified so that it's used here also - it may be useful in the future to have the data after it's been sent for hashing etc
dddfcbb7a0f844c173bab66e64e57675dfdd7cc2